easybuild/easyconfigs/v/VTune/VTune-2018_update1.eb

```diff
@@ -0,0 +1,15 @@
+name = 'VTune'
+version = '2018_update1'
+
+homepage = 'http://software.intel.com/en-us/intel-vtune-amplifier-xe'
+description = """Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,
+ Assembly and Java."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['vtune_amplifier_%(version)s.tar.gz']
+
+license_file = '12345@license.example.com'
```

Comment:
We consistently use `HOME + '/licenses/intel/license.lic'` here, which you can easily customise locally by defining `$INTEL_LICENSE_FILE` in the environment, or by editing the easyconfig file.

Do you mind changing this to `HOME + '/licenses/intel/license.lic'` so it's consistent with existing easyconfigs for Intel tools?

Reply:
I did this on purpose because we are using the license server and I have searched quite some time to find out how to change from a license file to a license server.
We should either put it in the files as a comment or have at least one sample config with this config.
There is really no good description on how to use the different license options for the intel block. 
If you want I can change it but we should really document it somehow.

Reply:
@Quarkmus I fully agree, this isn't clearly documented yet, indeed.

I think a comment that mention that this can be tweaked without changing the easyconfig file could be included above this line.

In addition, we should probably put together a dedicated section/page on this in the EasyBuild documentation at http://easybuild.readthedocs.io .

Would that be sufficient, do you think?

Also, would you have time to contribute to this?

Reply:
I'm really not an expert but I will try to help.
I will change the config tomorrow and look into the documentation.

Reply:
Thanks a lot @Quarkmus, much appreciated!

Do let us know if you have any questions, the easiest way may be via Slack (request an invitation via https://easybuild-slack.herokuapp.com/, join via https://easybuild.slack.com).